src/rascal_utility.hh

```diff
@@ -148,6 +148,13 @@ namespace rascal {
       return enumValue(e1) + enumSize<Enum1>() * enumValue(e2);
     }
 
+    template <typename Enum1, typename Enum2, typename Enum3>
+    constexpr size_t combineEnums(Enum1 e1, Enum2 e2, Enum3 e3) {
```

Comment:
this needs some documentation on what it is doing and why

Reply:
added documentation and tests